object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,9 @@
+/* exported defaults */
+function defaults(target, source) {
+  if (target.hasOwnProperty("bar") === true && source.hasOwnProperty("bar") === true && target.hasOwnProperty("baz") === true && source.hasOwnProperty("baz") === true && target.hasOwnProperty("foo") === true && source.hasOwnProperty("foo") === true) {
```

Comment:
I would try to do this another way. It is usually best to avoid such specific conditionals and allow for a more dynamic range of inputs